pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -115,6 +117,25 @@ func (d *dpiComponent) Objects() (objsToCreate, objsToDelete []client.Object) {
 			d.dpiDaemonset(),
 		)
 	}
+	// We always want to create these permissions when a management
+	// cluster is configured to allow any DPI running inside a
```

Comment:
nitty, but you're missing a comma here:

"when a management cluster is configured, to allow any DPI"

has a different meaning than:

"when a management cluster is configured to allow any API"

Reply:
Also nitty, but this comment should be within the first branch of the if / elseif / else

Reply:
Logic looks good though :+1: